tenets/codelingo/effective-go/package-comment/codelingo.yaml

```diff
@@ -0,0 +1,32 @@
+tenets:
+  - name: package-comment
+    flows:
+      codelingo/docs:
+        title: Package comment
+        body: |
+          Every package should have a package comment, a block comment preceding the package clause. 
+          For multi-file packages, the package comment only needs to be present in one file, and any one will do. 
+          The package comment should introduce the package and provide information relevant to the package as a 
+          whole. It will appear first on the godoc page and should set up the detailed documentation that follows.
+      codelingo/review:
+        comment: |
+          Every package should have a package comment, a block comment preceding the package clause.
+          For multi-file packages, the package comment only needs to be present in one file, and any one will do.
+    query: |
+      import codelingo/ast/go
+
+      @review comment
+      go.file(depth = any):
+        go.ident:
+          name as packageName
+        exclude:
+          go.comment(depth = any):
+            start_offset == 0
+      exclude:
+        go.file:
```

Comment:
Just noticed that the first go.file has (depth = any) but this go.file does not have a depth specified. I think we need to assert that both `go.file`s have the same depth. The best way to do that is probably to remove the (depth = any) from the go.file and put a parent go.dir(depth = any). Leaving it as it is will probably make the query as a whole match every file, which is bad and adding (depth = any) to this go.file is probably going to make compare all the files with each other which might be bad....